### PR TITLE
fix(input-field): prevent helper line to occupy screen space when hidden 

### PR DESCRIPTION
**Pull request by Kiarokh**
## Review:
- [ ] Commits are [atomic](https://seesparkbox.com/foundry/atomic_commits_with_git)
- [ ] Commits have the correct *type* for the changes made
- [ ] Commits with *breaking changes* are marked as such

### Browsers tested:
(Check any that applies, it's ok to leave boxes unchecked if testing something didn't seem relevant.)

Windows:
- [ ] Chrome
- [ ] Edge
- [ ] Firefox

Linux:
- [ ] Chrome
- [ ] Firefox

macOS:
- [ ] Chrome
- [ ] Firefox
- [ ] Safari

Mobile:
- [ ] Chrome on Android
- [ ] iOS

### CODE REVIEW

**Review by LucyChyzhova — CHANGES_REQUESTED** [review]
It doesn’t work if an `Input Field` has auto Completions - it covers the helper line

File: src/components/input-field/input-field.tsx [hunk 1]
```diff
@@ -486,8 +484,13 @@ export class InputField {
             return;
         }
 
+        const hideHelperLine = !(this.isFocused || this.isInvalid());
+
         return (
             <limel-helper-line
+                class={{
+                    hide: hideHelperLine,
```
[the remark this answers is not in the document]
Comment (LucyChyzhova) [reply src/components/input-field/input-field.tsx:492]: yes, I see your point! agree

File: src/components/input-field/input-field.tsx [hunk 2]
```diff
@@ -486,8 +484,13 @@ export class InputField {
             return;
         }
 
+        const hideHelperLine = !(this.isFocused || this.isInvalid());
+
         return (
             <limel-helper-line
+                class={{
```
[the remark this answers is not in the document]
Comment (Kiarokh) [reply src/components/input-field/input-field.tsx:491]: fixed here: https://github.com/Lundalogik/lime-elements/pull/2587

**Review by LucyChyzhova — DISMISSED** [review]
🚀

**Comment by LucyChyzhova** [comment]
We should wait to merge this PR until we implement this logic in other places. So the behaviour is consistent

**Comment by Kiarokh** [comment]
gonna do it using a CSS only solution: https://github.com/Lundalogik/lime-elements/pull/2591/

**Pull request closed without merging**